monitoring/src/main/scala/akkaviz/server/WebSocketRepl.scala

```diff
@@ -0,0 +1,178 @@
+package akkaviz.server
+
+import java.io.{InputStream, OutputStream, PrintStream}
+import java.util.concurrent.Executors
+
+import akka.http.scaladsl.model.ws._
+import akka.http.scaladsl.server.{Directives, Route}
+import akka.stream.scaladsl._
+import akka.util.ByteString
+import ammonite.ops.Path
+import ammonite.repl._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+import scala.concurrent.blocking
+
+trait WebSocketRepl {
+
+  def replPredef: String
+
+  def replArgs: Seq[Bind[_]]
+
+  def defaultReplPredef =
+    """
+      |import Predef.{println => _}
+      |import pprint.{pprintln => println}
+    """.stripMargin
+
+  implicit val ec = ExecutionContext.fromExecutor(Executors.newFixedThreadPool(4))
+
+  def replWebSocket: Route = {
+
+    def runRepl(replServerClassLoader: ClassLoader, in: InputStream, out: OutputStream) = {
+      val homePath = Path(Path.makeTmp)
+      val sshOut = new SshOutputStream(out)
+
+      Future {
+        val replSessionEnv = Environment(replServerClassLoader, in, sshOut)
+        Environment.withEnvironment(replSessionEnv) {
+          try {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/lustefaniak/akka-viz/file/2101331339/issues/source?bid=3121282&fileBranchId=3142893#l40)